### ISSUE TITLE
Traffic tile not available while starting the Valhalla server on Ubuntu

### ISSUE THREAD

**Issue by rushour123** [valhalla/valhalla#4212]
I successfully installed the Valhalla Server locally on Ubuntu and loaded data for Essex, UK, by following the steps provided in https://gis-ops.com/valhalla-part-1-how-to-install-on-ubuntu/ and https://gis-ops.com/valhalla-part-2-how-to-run-valhalla-on-ubuntu/. However, during the process of starting the Valhalla server, I encountered warning messages indicating that the "Traffic-tile extract could not be loaded."  This issue arose due to the absence of traffic-tile files, as shown in the screenshot. Instead of one thread I also used 2 to 4 threads but still faced the same issue. How to resolve this and utilize the routing service effectively. 
![image](https://github.com/valhalla/valhalla/assets/140007801/4efe6a16-2c90-454b-962c-afd3b6c815d8)

**Comment by nilsnolde** [comment]
Oh that's a big topic and doesn't fit into an issue. Search for "live traffic" in the issues and discussions, there's many things giving pointers and hints. Also on the homepage you're referencing I published a blog post how traffic works in Valhalla: https://gis-ops.com/traffic-in-valhalla/. Though that's pretty old, and might not've been a time when I really knew what I was talking about, I should probably update it.

In short: no worries, you're only missing out on a nice but pretty expensive feature, like 99.99% of the Valhalla server instances ever started:)
